taskcluster/scripts/write-dummy-secret.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
This is a copy of https://github.com/mozilla-mobile/firefox-ios/blob/main/taskcluster/scripts/write-dummy-secret.py